src/components/css/subdomains.scss

```diff
@@ -104,44 +111,100 @@ $page-margin: 180px;
         margin-top: 5px;
         margin-bottom: 5px;
         margin-right: 30px;
+
         position: relative;
         float: left;
-        width: 366px;
-        height: 178px;
+        width: 295px;
```

Comment:
Do we need to use static pixels heights on this element? Normally that could limit its responsiveness. If so, would it be possible to use max/min-width/height instead?